src/main/scala/org/antipathy/scoozie/action/Hive2Action.scala

```diff
@@ -0,0 +1,145 @@
+/**
+  *    Copyright (C) 2019 Antipathy.org <support@antipathy.org>
+  *
+  *    Licensed under the Apache License, Version 2.0 (the "License");
+  *    you may not use this file except in compliance with the License.
+  *    You may obtain a copy of the License at
+  *
+  *        http://www.apache.org/licenses/LICENSE-2.0
+  *
+  *    Unless required by applicable law or agreed to in writing, software
+  *    distributed under the License is distributed on an "AS IS" BASIS,
+  *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  *    See the License for the specific language governing permissions and
+  *    limitations under the License.
+  */
+package org.antipathy.scoozie.action
+
+import com.typesafe.config.Config
+import org.antipathy.scoozie.action.prepare.Prepare
+import org.antipathy.scoozie.builder.{ConfigurationBuilder, HoconConstants, MonadBuilder, PrepareBuilder}
+import org.antipathy.scoozie.configuration._
+import org.antipathy.scoozie.exception.ConfigurationMissingException
+
+import scala.collection.JavaConverters._
+import scala.collection.immutable._
+import scala.xml.Elem
+
+/**
+  * Oozie Hive2 action definition
+  *
+  * @param name the name of the action
+  * @param scriptName the name of the hive script
+  * @param scriptLocation the path to the hive script
+  * @param jdbcUrl the Jdbc url to connect to Hive2 server from Beeline, required
+  * @param parameters a collection of parameters to the hive script
+  * @param jobXmlOption optional job.xml path
+  * @param files additional files to pass to job
+  * @param configuration additional config for this action
+  * @param yarnConfig Yarn configuration for this action
+  * @param prepareOption an optional prepare stage for the action
+  */
+final class Hive2Action(override val name: String,
```

Comment:
Could you update the *.md files with this action?

Reply:
Good call. Will do! I had forgotten this PR was still open when I pushed the Hive2 action changes here.